ext/opentelemetry-ext-requests/src/opentelemetry/ext/requests/__init__.py

```diff
@@ -55,7 +55,7 @@
 
 
 # pylint: disable=unused-argument
-def _instrument(tracer_provider=None):
+def _instrument(tracer_provider=None, apply_custom_attributes_on_span=None):
```

Comment:
technically, this can do any operation on an existing span, rather than just set attributes.

Thoughts on calling this something more general? like "span_callback"

Reply:
`span_callback` sounds good. Done.